scss/typography/_print.scss

```diff
@@ -77,5 +77,10 @@ $print-hrefs: true !default;
     // Avoid page breaks after a heading
     h2,
     h3 { page-break-after: avoid; }
+
+    // Helper to re-allow page breaks in the middle of certain elements (e.g. pre, blockquote, tr)
+    .print-break-inside {
```

Comment:
`.print-page-break` makes me think I can use that on an element anywhere I want a page break in print (e.g. in-between two <p>s), but that's not what this feature does.

`.print-break-inside` on a containing element makes more sense to me, semantically.